tests/lang-c.yaml

```diff
@@ -12,3 +12,9 @@
   command: [wc,/etc/apt/sources.list.d/llvm.list]
   assert:
   - status == 0
+- desc: it should have major version
+  command: [npx envinfo]
```

Comment:
npx comes from Node.js, looks not available on lang-c. Maybe use `clang --version`

Reply:
Aye, on it!

Reply:
Fixed in [c767edd](https://github.com/gitpod-io/workspace-images/pull/1377/commits/c767edd446fdd058f0d9a4e0d313bf9676a711fb)